bitrise.yml

```diff
@@ -48,7 +48,6 @@ stages:
     - ui-tests-3: {}
     - ui-tests-4: {}
     - integration-all: {}
-    - deploy-dry-run: {}
```

Comment:
This also removes it from `releases-trigger-pipeline`. Are we okay with that?

I'm guessing this is okay given that we would be manually executing the deploy for releases, but it would mean that we wouldn't catch any breakages until we attempt to run the deploy scripts as opposed to catching them earlier in CI.

Reply:
I think it was never running in `releases-trigger-pipeline`, it was only running in the main pipeline. We should generally catch release breakages in the nightly build. (And even those are unlikely, as the other builds should catch most breakages)